### PR TITLE
Submission changes

### PR DESCRIPTION
**Pull request by addy419**

### CODE REVIEW

File: Indico/Mutation/SubmitReview.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            vars.changes = Changes.ToString();
+            vars.rejected = Rejected;
+
+            if (ForceComplete != null)
+            {
+                args.Add("forceComplete", "Boolean");
+                vars.forceComplete = ForceComplete;
+            }
+
+            string queryArgs = string.Join(", ", args.Select(pair => $"${pair.Key}: {pair.Value}"));
```
Comment (rkaczka-vd) [inline Indico/Mutation/SubmitReview.cs:46]: It's usually preferable to use `var` keyword;

**Review by rkaczka-vd — COMMENTED** [review]
There should be tests for new mutation.

File: Indico/Mutation/WorkflowSubmissionBase.cs [hunk 2]
```diff
@@ -32,16 +33,12 @@ public class WorkflowSubmissionBase : Mutation<JObject>
         /// Executes request and returns Job
         /// </summary>
         /// <returns>Job</returns>
-        async public Task<JObject> Exec()
+        public async Task<JObject> Exec()
         {
-            if (this.Files == null && this.Streams == null && this.Urls == null)
+            if (Convert.ToInt16(this.Files != null) + Convert.ToInt16(this.Streams != null) + Convert.ToInt16(this.Urls != null) != 1)
```
[the remark this answers is not in the document]
Comment (addy419) [reply Indico/Mutation/WorkflowSubmissionBase.cs:38]: I actually thought about it but ((a && !b && !c) || (!a && b && !c) || (!a && !b && c)) seemed complex to understand for such simple task.

File: Indico/Mutation/SubmitReview.cs (hunk 1 above)
Comment (addy419) [reply Indico/Mutation/SubmitReview.cs]: Fixed

**Review by rkaczka-vd — APPROVED** [review]

**Review by goatrocks — APPROVED** [review]

**Review by szymon-socha — APPROVED** [review]

**Pull request merged**